apps/server/src/modules/oauth-provider/controller/dto/response/consent-session.response.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ConsentSessionResponse {
+	constructor(response: ConsentSessionResponse) {
+		this.client_id = response.client_id;
+		this.client_name = response.client_name;
+		this.challenge = response.challenge;
+	}
+
+	@ApiProperty()
+	client_id?: string;
+
+	@ApiProperty()
+	client_name?: string;
+
+	@ApiProperty()
+	challenge?: string;
```

Comment:
Why are all props optional?

Reply:
We stick to the hydra documentation for the properties. There they were all marked as optional for whatever reason.